translate/src/utils/message/requiresSourceView.test.js

```diff
@@ -201,16 +201,4 @@ my-entry =
 
     expect(requiresSourceView(message)).toEqual(true);
   });
-
-  it('returns true for a string with an unsupported selector', () => {
-    const input = `
-my-entry =
-    { FOO() ->
-        [foo] FOO
-       *[other] BAR
-    }`;
-    const message = parseEntry(input);
-
-    expect(requiresSourceView(message)).toEqual(true);
-  });
```

Comment:
I guess dropping this validation will not cause too many headaches.

Reply:
Yeah, in practice we have no unknown selectors, and really we ought to show the selector in the message context.